smart-contract/contracts/Main.sol

```diff
@@ -101,6 +101,25 @@ contract ZuriSchoolVoting is VotingEvents, VotingAccess {
         emit StudentCreated(_name, _student);
     }
 
+    /**
+        * @notice add students to the system
+        * @dev add students to the system from a list of addresses and names
+    */
+    function addStudents(address[] memory _students, string[] memory _names) external isChairperson {
+        require(_names.length > 0, "0 names");
+        require(_students.length > 0, "0 addresses");
+        require(_students.length == _names.length, "names != addresses");
+
+        for(uint256 i = 0; i < _students.length; i++) {
+            require(bytes(voters[_students[i]].name).length == 0, "student already exist");
+            Voter memory _voter = Voter(_names[i], true, Stakeholder.STUDENT);
+            _addVoter(_students[i], _voter);
```

Comment:
I think we can re-use `addStudent` function here as it also includes `emitting of events`